analyses/molecular-subtyping-HGG/results/HGG_molecular_subtype.tsv

```diff
@@ -1,238 +1,238 @@
-PT_X648RVMK	7316-238	NA	BS_N6N147BY	HGG, To be classified	None documented	0.47585177684628965	0	0	0	0	NA	NA	NA	NA	0	0	Other
-PT_X71D7G5S	7316-1105	BS_P8WN8XEQ	BS_C6ZZ3FY5	HGG, H3 G35, TP53 loss	None documented	0.7137495389663795	2	0	0	0	p.S261Vfs*84, p.N311Kfs*24	NA	NA	NA	0	0	loss
-PT_X7HC5YCY	7316-255	NA	BS_FN07P04C	HGG, To be classified	None documented	0.24857708185766048	0	0	0	0	NA	NA	NA	NA	0	0	Other
-PT_X7HC5YCY	7316-255	NA	BS_W4H1D4Y6	HGG, To be classified	None documented	0.11125688161807704	0	0	0	0	NA	NA	NA	NA	0	0	Other
-PT_XKVFVBY4	7316-550	NA	BS_W4DB5RP1	HGG, To be classified	None documented	0.8683018696130294	0	0	0	0	NA	NA	NA	NA	0	0	Other
-PT_Y585X1BS	7316-622	NA	BS_W1385R8Y	HGG, To be classified	None documented	0.9723999347017674	0	0	0	0	NA	NA	NA	NA	0	0	Other
-PT_Y5KY6KN9	7316-195	BS_AYPBAHJ6	BS_4QW2S1Q8	DMG, H3 K28, TP53 loss	None documented	0.8834689847132025	1	0	0	0	p.X331_splice	NA	NA	NA	0	0	loss
-PT_Y5KY6KN9	7316-195	BS_JGKRN7NA	BS_MX23ZY0Y	DMG, H3 K28, TP53 loss	None documented	0.8972408459164949	1	0	0	0	p.X331_splice	NA	NA	NA	0	0	loss
-PT_Y5KY6KN9	7316-195	BS_XMP9XNR9	BS_P9JP6JFA	DMG, H3 K28, TP53 loss	None documented	0.957367513996152	1	0	0	0	p.X331_splice	NA	NA	NA	0	0	loss
-PT_Y660CKCB	7316-3122	BS_FKQ7F6D1	BS_XG0DBWVT	DMG, H3 K28, TP53 loss	None documented	0.9611065537230697	1	1	0	0	p.G266R	2	NA	NA	1	0	loss
-PT_Y74CVASJ	A16538	BS_KPG4WCHS	BS_2JP7RBMB	DMG, H3 K28, TP53 loss	NA	0.6855910279776566	2	0	0	0	p.M66Ifs*57, p.V173M	NA	NA	NA	1	0	loss
-PT_Y76A7PBA	7316-2288	BS_M4E4H6NG	BS_VVTHDRBD	HGG, H3 wildtype	None documented	0.8423935158971516	0	0	0	0	NA	NA	NA	NA	0	0	Other
-PT_Y8RNS9JE	7316-1656	BS_3BMGF6GN	BS_X23724NJ	HGG, H3 wildtype, TP53 activated	None documented	0.7859633910639032	1	0	0	0	p.R248L	NA	NA	NA	1	1	activated
-PT_Z4BF2NSB	7316-161	BS_W2QCHQ7E	BS_SHJA4MR0	HGG, H3 wildtype, TP53 loss	None documented	0.7716700140909606	2	0	0	0	p.R342P, p.P72Lfs*48	NA	NA	NA	1	0	loss
-PT_Z4BF2NSB	7316-161	BS_W2QCHQ7E	BS_X0XXN9BK	HGG, H3 wildtype, TP53 loss	None documented	0.4490920924301138	2	0	0	0	p.R342P, p.P72Lfs*48	NA	NA	NA	1	0	loss
-PT_Z4PJA6KT	7316-1763	BS_853PNV7P	BS_DRY58DTF	DMG, H3 K28, TP53 loss	None documented	0.6103712256539268	1	0	0	0	p.R213*	NA	NA	NA	0	0	loss
-PT_Z4PJA6KT	7316-1763	BS_TF5TTEXH	BS_E60JZ9Z3	DMG, H3 K28, TP53 loss	None documented	0.903330102831969	1	0	0	0	p.R213*	NA	NA	NA	0	0	loss
-PT_Z4PJA6KT	7316-1763	BS_WPCBK1EG	BS_6WP1FHTE	DMG, H3 K28, TP53 loss	None documented	0.8810750708629667	1	0	0	0	p.R213*	NA	NA	NA	0	0	loss
-PT_ZVYRFGEN	7316-1068	NA	BS_ZV1P6W9C	HGG, To be classified	None documented	0.8801571782954959	0	0	0	0	NA	NA	NA	NA	0	0	Other
+Kids_First_Participant_ID	sample_id	Kids_First_Biospecimen_ID_DNA	Kids_First_Biospecimen_ID_RNA	molecular_subtype	cancer_predispositions	tp53_score	tp53_SNV_indel_counts	tp53_CNV_loss_counts	SV_counts	tp53_HGVSp_Short	tp53_CNV_loss_evidence	SV_type	tp53_hotspot	tp53_activating	tp53_altered
```

Comment:
Do we need an update and rerun of the tp53 module? Seems to be missing the Fusion_counts and Fusion_evidence columns now.

Reply:
@kgaonkar6, the tp53 module was updated and re-ran[ in this PR](https://github.com/PediatricOpenTargets/OpenPedCan-analysis/pull/94) . But note that this is re-run with only PBTA samples. 
I will look into the `Fusion_counts` and `Fusion_evidence` column.

Reply:
After some digging figure out what the issue is - so when I started modifying the `tp53-nf1-score` module, I did not pull in the most up-to-date changes in OpenPBTA and there are updates that added the fusion columns. That's why we don't have this. 
I will create a ticket to add additional changes in the module - but in the meantime, for this PR, since that column is not required for HGG subtyping, maybe we should leave it as is for now and then re-run after the tp53 module is updated?